doc/tutorials/gapi/face_beautification/face_beautification.markdown

```diff
@@ -0,0 +1,500 @@
+# Implementing a face beautification algorithm on G-API {#tutorial_gapi_face_beautification}
+
+@prev_tutorial{tutorial_gapi_anisotropic_segmentation}
+
+[TOC]
+
+# Introduction {#gapi_fb_intro}
+
+In this tutorial you will learn:
+* How to implement a custom kernel (using OpenCV backend);
+* How to use Inference-API to infer networks inside the graph;
```

Comment:
There's no "Inference-API", and there's no "Streaming-API" as well. Both are just new _features_ of "G-API"